### PR TITLE
Fix arguments in various ABORT messages

### PR DESCRIPTION
**Pull request by fejoa**

### CODE REVIEW

File: Main/Source/devcons.cpp [hunk 1]
```diff
@@ -300,7 +300,7 @@ void devcons::AddDevCmd(festring fsCmd, callcmd Call, festring fsHelp,bool bWiza
 {
   callcmd cc = Find(fsCmd.CStr());
   if(cc!=NULL) //TODO ignore if equal?
-    ABORT("command %s already set %x %x",fsCmd.CStr(),cc,Call);
+    ABORT("command %s already set %x %x",fsCmd.CStr(),(uint)cc,(uint)Call);
```
Comment (iology) [inline Main/Source/devcons.cpp]: Should use the `%p` specifier, no need to cast.

Comment (fejoa) [reply Main/Source/devcons.cpp]: Awesome, thanks @iology

File: Main/Source/bugworkaround.cpp [hunk 2]
```diff
@@ -277,8 +277,8 @@ bool bugfixdp::ScanLevelForCharactersAndItemsWork(
       ABORT( //more than 2 copies (one valid and one invalid) is more complicated and the code is not ready yet to deal with that.
         "iDupCount=%d found for item 0x%x id=%lu."
         "tot=%d item pointer/object/reference 0x%x found for id=%lu.",
-        iDupIDCount,(uint)itWork,itWork->GetID(),
-        iPointerMatchCount,(uint)itWork,itWork->GetID()
+        iDupIDCount,reinterpret_cast<uint*>(itWork),itWork->GetID(),
```
Comment (fejoa) [inline Main/Source/bugworkaround.cpp]: @iology do you think this is an appropriate treatment of `itWork` here?

Comment (iology) [reply Main/Source/bugworkaround.cpp]: Pointers use `%p`. You can replace `0x%x` with `%p`. The output is also like "0x10d448f60". No cast needed.

**Comment by iology** [comment]
What are LGTM's compiler flags?

**Comment by fejoa** [comment]
> What are LGTM's compiler flags?

I'm not particularly sure. There's such thing as an [lgtm project configuration file](https://lgtm.com/help/lgtm/lgtm.yml-configuration-file) that offers some control over configurations, but I don't know exactly what level of stringency their warnings go to.

**Comment by iology** [comment]
I forgot to squash the commits. :p

**Pull request merged**